src/components/cards/BldgCardImage.tsx

```diff
@@ -1,45 +0,0 @@
-import React from "react";
-
-// images this component may need to use -- is there a better way?
-import archeryHut from "../../assets/images/archeryHut-01.jpg";
-import healingChamber from "../../assets/images/healingChamber-01.jpg";
-import mealHall from "../../assets/images/mealHall-01.jpg";
-import swordsmithy from "../../assets/images/swordsmithy-01.jpg";
-import townCenter from "../../assets/images/townCenter-01.jpg";
-import { BuildingType } from "../../types";
```

Comment:
So you could create an `images.ts` file that does all these exports and then re-exports them for you.